docs/extensibility-http.md

```diff
@@ -179,7 +179,7 @@ https://github.com/[[GITHUB-USER]]?tab=packages
 ## Deploy Device(s) & Discovery
 
 ```bash
-git clone https://github.com/DazWilkin/akri-http
+git clone --branch http-extensibility https://github.com/DazWilkin/akri-http
```

Comment:
the repo should be github.com/deislabs/akri

Reply:
The spoofed devices and discovery repo is actually distinct `https://github.com/DazWilkin/akri-http`

One thing that I should perhaps do is merge this into `./samples/broker/http`?

I created it as a standalone (use your preferred) example.

Thoughts?

Reply:
as a nessie replacement, my preference is to have all the code collocated in a single branch somehow.

Reply:
Fixed: https://github.com/deislabs/akri/pull/135/commits/8865707f2bf01cee75e4240cc7a0b90dc4ff8974

Reply:
Updated documentation: https://github.com/deislabs/akri/pull/135/commits/64ae96c97dbca0897cf71c04de1b8fc026c47d47